### PR TITLE
Make full site

### PR DESCRIPTION
**Pull request by BonkeGcobo**
1) Added React router to add multiple pages.
2) Add home page.
3) Add about page

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <div className="homeText">
+      <p className="Text1">
+        lorem ipsum odor amet, consectetuer adipiscing elit.
+        Ac purus in massa egestas mollis varius;
+        dignissim elementum. Mollis tincidunt mattis hendrerit dolor eros enim, nisi ligula ornare.
+        Hendrerit parturient habitant pharetra rutrum gravida porttitor eros feugiat. Mollis elit
+        sodales taciti duis praesent id. Consequat urna vitae morbi nunc congue.
+        lorem ipsum odor amet, consectetuer adipiscing elit.
+        Ac purus in massa egestas mollis varius;
+        dignissim elementum. Mollis tincidunt mattis hendrerit dolor eros enim, nisi ligula ornare.
```
Comment (micheaol) [inline src/components/Home.js:17]: Please ensure that the general layout should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).

Your Implementation | Figma Design
:---:|:---:
image | image


Your Implementation | Figma Design
:---:|:---:
image | image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @BonkeGcobo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/App.css [hunk 2]
```diff
@@ -84,8 +84,9 @@ body {
 }
 
 .calSection {
+  margin-top: 1.5rem;
   display: flex;
-  gap: 1rem;
+  height: 100vh;
```
Comment (Amrhub) [inline src/App.css]: - [x] I appreciate your work on getting the layout to match the wireframe but if you noticed we have to scroll an empty space to see the footer. I went ahead and investigated a solution that you might like. we could just give a general style to our #root `height: 100vh` then set a height percentage for section for example `80%` and header `10%` footer `10%`. that would look neat for this given webiste

File: src/App.css [hunk 3]
```diff
@@ -111,10 +112,9 @@ a:hover {
 }
 
 .Home-section {
-  text-align: center;
   padding-top: 1rem;
   background-color: #e0e0e0;
-  height: 80%;
+  height: 100vh;
```
Comment (Amrhub) [inline src/App.css]: Kind remainder 😇

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @BonkeGcobo, 👋🏻 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight ✔️ 
Excellent communication with previous code reviewer 😇 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 3 above)
Comment (BonkeGcobo) [reply src/App.css]: Thank you very much for your suggestions

**Review by julie-ify — APPROVED** [review]
Hi @BonkeGcobo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**